sql_generators/active_users_aggregates_v3/templates/mobile_checks.sql

```diff
@@ -57,3 +57,33 @@ SELECT
     ),
     NULL
   );
+
+{% raw -%}
```

Comment:
Is this `raw` in the correct place? I'm not familiar with it's use,  it it seems different from the other files.

Reply:
In this case `raw` is used to indicate that the text between this and the corresponding end block should not be modified by jinja when generating the output of this template. We have to do this in some cases, for example in this case we generate another jinja template so we want to preserve some of the template features like:

```
`{{ project_id }}.{{ dataset_id }}.{{ table_name }}`
```